dev/workflow/rocoto/gcafs_tasks.py

```diff
@@ -1038,11 +1038,11 @@ def metp(self):
                 dep_dict = {'type': 'taskvalid', 'name': f'{self.run}_arch_vrfy', 'condition': 'not'}
                 deps2.append(rocoto.add_dependency(dep_dict))
                 for lookback2 in range(1, lookback):
-                    offset = timedelta_to_HMS(-to_timedelta(f'{6*lookback2}H'))
+                    offset = timedelta_to_HMS(-to_timedelta(f'{6 * lookback2}H'))
```

Comment:
Should `6` be changed to `assim_freq`?

Reply:
Yeah, I noticed multiple instances after this comment.  Let's hold off for now.  We should do a sweep/refactor of the workflow setup scripts to allow for other `assim_freq`s at some point, but not in this PR.